### DISCUSSION TITLE
snapshot and update

### DISCUSSION THREAD

**Discussion by robertocannito** [SpiralLinux/SpiralLinux-project#28] [General]
Hi, for gnome de what is the correct way for restore snapshot?
what is correct way for update or upgrade?

Thank you since now.

Rob.

**Comment by geckolinux** [comment]
Hi there Rob,

> what is the correct way for restore snapshot?

https://github.com/SpiralLinux/SpiralLinux-project/wiki#working-with-btrfs-snapshots-and-rollbacks

> what is correct way for update or upgrade?

https://github.com/SpiralLinux/SpiralLinux-project/wiki#updating-and-upgrading-spirallinux

**Reply by robertocannito** [reply]
Than you Sam,
to you plan also to include BTRFS-Assistant?
Should be good.

Thanks a lot,

Rob.

**Reply by geckolinux** [reply]
Hi Rob,
No, I looked at it, but there are no packages available for Debian, and it requires very heavy Python dependencies and it doesn't work well on Debian.
